Projects/UOContent/Gumps/ConfirmHouseResize.cs

```diff
@@ -3,170 +3,171 @@
-                        }
+                    if (toGive != null)
+                    {
+                        var box = _mobile.BankBox;
 
-                        if (toGive != null)
+                        if (box.TryDropItem(_mobile, toGive, false))
                         {
-                            var box = m_Mobile.BankBox;
-
-                            if (box.TryDropItem(m_Mobile, toGive, false))
+                            if (toGive is BankCheck check)
                             {
-                                if (toGive is BankCheck check)
-                                {
-                                    m_Mobile.SendLocalizedMessage(
-                                        1060397,
-                                        check.Worth.ToString()
-                                    ); // ~1_AMOUNT~ gold has been deposited into your bank box.
-                                }
-
-                                m_House.RemoveKeys(m_Mobile);
-                                new TempNoHousingRegion(m_House, m_Mobile);
-                                m_House.Delete();
-                            }
-                            else
-                            {
-                                toGive.Delete();
-                                m_Mobile.SendLocalizedMessage(500390); // Your bank box is full.
+                                _mobile.SendLocalizedMessage(
+                                    1060397,
+                                    check.Worth.ToString()
+                                ); // ~1_AMOUNT~ gold has been deposited into your bank box.
                             }
+
+                            _house.RemoveKeys(_mobile);
+                            new TempNoHousingRegion(_house, _mobile);
+                            _house.Delete();
                         }
                         else
                         {
-                            m_Mobile.SendMessage("Unable to refund house.");
+                            toGive.Delete();
+                            _mobile.SendLocalizedMessage(500390); // Your bank box is full.
                         }
                     }
-                }
-                else
-                {
-                    m_Mobile.SendLocalizedMessage(501320); // Only the house owner may do this.
+                    else
+                    {
+                        _mobile.SendMessage("Unable to refund house.");
+                    }
                 }
             }
-            else if (info.ButtonID == 0)
+            else
             {
-                m_Mobile.CloseGump<ConfirmHouseResize>();
-                m_Mobile.SendGump(new HouseGumpAOS(HouseGumpPageAOS.Customize, m_Mobile, m_House));
+                _mobile.SendLocalizedMessage(501320); // Only the house owner may do this.
             }
         }
+        else if (info.ButtonID == 0)
+        {
+            _mobile.CloseGump<ConfirmHouseResize>();
```

Comment:
I think this is redundant, cause at this point, the gump is closed (due to the response).